exonum-node/src/sandbox/tests/basic.rs

```diff
@@ -234,12 +234,12 @@ fn test_retrieve_block_and_precommits() {
     assert!(bl_proof_option.is_some());
     let block_proof = bl_proof_option.unwrap();
     let block = block_proof.block;
-    let precommits: Vec<Verified<Precommit>> = block_proof.precommits;
     let expected_height = target_height.previous();
     let expected_block_hash = block.object_hash();
 
     assert_eq!(expected_height, block.height);
-    for precommit in precommits {
+    for precommit in block_proof.precommits {
+        let precommit: Verified<Precommit> = precommit.into_verified().unwrap();
```

Comment:
If we want to test here that signatures are correct, I think it worth to use `expect`.
Otherwise, `try_from` will be enough.

Same for other similar places in tests.